### DISCUSSION TITLE
Is the new release gonna come out soon?

### DISCUSSION THREAD

**Discussion by isle9** [RafaelGB/obsidian-db-folder#171] [Q&A] [answered]
@RafaelGB I see that you've added a lot of new stuff. I'm guessing that it should be released soon?

**Comment by RafaelGB** [comment, marked as answer]
I am currently on my holidays period, so I am not so active than I used to be.
I will try to release the version next week :)

**Comment by RafaelGB** [comment]
https://github.com/RafaelGB/obsidian-db-folder/releases/tag/1.8.0 was released =)
By the flys with the incorporation to the work I wouldn't have had time
